mkudffs/main.c

```diff
@@ -41,8 +41,12 @@
 #include <limits.h>
 #include <dirent.h>
 #include <sys/ioctl.h>
+#if defined(__linux__)
 #include <linux/fs.h>
 #include <linux/fd.h>
+#elif defined(__APPLE__)
+#include <sys/disk.h>
```

Comment:
Please add comment for which function (or macro?) is this sys/disk.h include file needed. I really do not know.

Reply:
Sure thanks, I'll add a comment straight away. It's the Darwin equivalent of fs.h (and also tackles fd.h).